python/create_service_diagnostics_bundle.py

```diff
@@ -135,6 +135,14 @@ def parse_args() -> dict:
         help="Disable interactive mode and assume 'yes' is the answer to all prompts.",
     )
 
+    parser.add_argument(
+        "--service_diagnostics_version",
```

Comment:
The other arguments are in [Kebab case](http://wiki.c2.com/?KebabCase).

Reply:
Argument removed, file VERSION added because of this do not need send info via arguments